### PR TITLE
added extra cell for status (passed/failed)

### PR DESCRIPTION
**Pull request by amavinag**
*  added another cell for the table row with
     status passed/failed.

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 59094ac2d7c7e1553d798b4380988ec69588fbe8:

Everything seems all right :smile: .
*CFME QE Bot*

File: data/templates/template_tester_report.html [hunk 1]
```diff
@@ -30,6 +30,7 @@
                           style="line-height: 1.66666667em; padding: 0px 8px;">
                     </span>{{passed}}
                 </td>
+                <td class="success"> passed </td>
```
Comment (psav) [inline data/templates/template_tester_report.html]: I think you'll need a new column header too right?

Comment (amavinag) [inline data/templates/template_tester_report.html]: psav: yes, added 'Result' column header.

**Comment by dajoRH** [comment]
Lint report for commit 835c766fd300fd198be9c1ea17abb24d6d46458f:

Everything seems all right :smile: .
*CFME QE Bot*

**Pull request merged**